android/app/build.gradle

```diff
@@ -113,4 +113,8 @@ dependencies {
     releaseCompile 'com.squareup.leakcanary:leakcanary-android-no-op:1.3.1' // or 1.4-beta1
     testCompile 'com.squareup.leakcanary:leakcanary-android-no-op:1.3.1' // or 1.4-beta1
 
+    // For UI improvements
+    compile 'me.zhanghai.android.materialprogressbar:library:1.1.7'
+    compile 'com.balysv:material-ripple:1.0.2'
```

Comment:
Also please try not to use any external library unless absolutely necessary.
We do not want to increase the number of dependencies.
Try to implement UI changes w/o these libraries